src/V3Stats.cpp

```diff
@@ -74,12 +91,36 @@ class StatsVisitor final : public VNVisitor {
         }
     }
 
+    void addRAMStats(AstNodeModule* nodep) {
+        for (const auto& nodep : m_potentialRams) {
+            AstVar* varp = nodep->varp();
+            AstVarScope* varScopep = nodep->varScopep();
+            int depth = varp->dtypep()->arrayUnpackedElements();
+            int bits = depth * varp->width();
```

Comment:
```suggestion
            const AstVar* varp = nodep->varp();
            const AstVarScope* varScopep = nodep->varScopep();
            const int depth = varp->dtypep()->arrayUnpackedElements();
            const int bits = depth * varp->width();
```

Perhaps other const's needed elsewhere.